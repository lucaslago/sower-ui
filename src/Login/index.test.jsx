import { Login } from './index';

describe('<Login />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper.length).toEqual(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

