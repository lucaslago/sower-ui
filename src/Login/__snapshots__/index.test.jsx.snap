exports[`<Login /> renders without crashing 1`] = `
<Row
  bsClass="row"
  componentClass="div">
  <Col
    bsClass="col"
    componentClass="div"
    md={4}
    mdOffset={4}
    sm={6}
    smOffset={3}
    xs={12}>
    <Paper
      circle={false}
      rounded={true}
      style={
        Object {
          "marginTop": "2rem",
          "paddingBottom": "3rem",
          "paddingLeft": "3rem",
          "paddingRight": "3rem",
          "paddingTop": "0rem",
        }
      }
      transitionEnabled={true}
      zDepth={2}>
      <form
        onSubmit={[Function]}
        style={
          Object {
            "marginTop": "5rem",
          }
        }>
        <Row
          bsClass="row"
          componentClass="div">
          <Subheader
            inset={false}
            style={
              Object {
                "fontSize": "20px",
                "paddingLeft": "0px",
                "textAlign": "center",
              }
            }>
            Sign In
          </Subheader>
        </Row>
        <Row
          bsClass="row"
          componentClass="div">
          <ValidatedTextField
            errorText="E-mail is required"
            handleChange={[Function]}
            label="E-mail"
            showValidationError={false}
            type="email" />
        </Row>
        <Row
          bsClass="row"
          componentClass="div">
          <ValidatedTextField
            errorText="Password is required"
            handleChange={[Function]}
            label="Password"
            showValidationError={false}
            type="password" />
        </Row>
        <Row
          bsClass="row"
          componentClass="div"
          style={
            Object {
              "marginTop": "2rem",
              "textAlign": "center",
            }
          }>
          <AsyncButton
            label="login"
            makingRequest={false}
            type="submit" />
        </Row>
      </form>
    </Paper>
  </Col>
  <Snackbar
    autoHideDuration={4000}
    message="Incorrect email and/or password"
    onRequestClose={[Function]}
    open={false}
    style={
      Object {
        "textAlign": "center",
      }
    } />
</Row>
`;
